examples/light-switch-app/silabs/efr32/src/AppTask.cpp

```diff
@@ -99,11 +100,11 @@ void OnTriggerIdentifyEffect(Identify * identify)
     ChipLogProgress(Zcl, "Trigger Identify Effect");
     sIdentifyEffect = identify->mCurrentEffectIdentifier;
 
-    if (identify->mCurrentEffectIdentifier == EMBER_ZCL_IDENTIFY_EFFECT_IDENTIFIER_CHANNEL_CHANGE)
+    if (identify->mCurrentEffectIdentifier == Clusters::Identify::EffectIdentifierEnum::kChannelChange)
     {
         ChipLogProgress(Zcl, "IDENTIFY_EFFECT_IDENTIFIER_CHANNEL_CHANGE - Not supported, use effect varriant %d",
-                        identify->mEffectVariant);
-        sIdentifyEffect = static_cast<EmberAfIdentifyEffectIdentifier>(identify->mEffectVariant);
+                        (uint8_t) identify->mEffectVariant);
+        sIdentifyEffect = static_cast<Clusters::Identify::EffectIdentifierEnum>(identify->mEffectVariant);
```

Comment:
Again, does not make sense.  Looks like this weird code got copy/pasted.  :(  I'll stop commenting on it, but please collect up the instances in the issue you file.